build-scripts/copyNonCodeFiles.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/*
+    This script is called from npm run compile. It copies the
+    files and directories listed in `relativePaths` to `out/`.
+*/
+
+const fs = require('fs-extra');
+const path = require('path');
+
+const repoRoot = path.dirname(__dirname);
+const outRoot = path.join(repoRoot, 'out');
+
+// May be individual files or entire directories.
+const relativePaths = [
+    path.join('src', 'schemas')
+];
+
+(async () => {
```

Comment:
Consider making this a proper function with proper params that could be called here and perhaps exported and tested.

e.g. ```
async function copyNonCodeFiles({sourcePath=repoRoot, destPath=outRoot}={}, {
   sourcePath: string,
   destPath: string
}) { /* ... */ }```

Reply:
For now I'm going to stick to the pattern used by `clean.js` and `bundleDeps.js`.